schema/schema_template.py

```diff
@@ -405,6 +408,53 @@ def validate_parallax_sig_figs(self, key, value):
 
         return value
 
+
+class RadialVelocities(Base):
+    # This is an example of a measurement table (see below, commented-out table as another example)
+
+    __tablename__ = 'RadialVelocities'
+    source = Column(
+        String(100),
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
+
+    # always note units, following the practices of Chen et al. 2022.
+    radial_velocity_km_s = Column(Float, nullable=False)
+
+    radial_velocity_error_km_s = Column(Float)  # todo: make asymmetric errors
```

Comment:
Do we want to include the option for asymmetric errors here?  If we do, now's the time!  I don't feel strongly one way or the other.

Reply:
yeah, I'll implement them!

Reply:
Upon further discussion, we decided not to implement this now.  (One datum: Gaia only provides a symmetric uncertainty for radial velocities.)